### DISCUSSION TITLE
License required for personal blog?

### DISCUSSION THREAD

**Discussion by thegrahamking** [statiqdev/Discussions#50] [General]
Hi,

I'm considering using Statiq for a personal blog (coding focused but all content written by me).  In this situation would I need a license? Would it make a difference if my blog hosted ads?

Many thanks

**Comment by daveaglick** [comment]
Thanks for asking for clarification!

> In this situation would I need a license?

No, a personal blog, portfolio, etc. would not need a license.

> Would it make a difference if my blog hosted ads?

This gets a little trickier. The license terms are intentionally simple ([flipped form](https://flippedform.com/)). The license states:

> This license allows you to use and share this software for noncommercial purposes for free

Which I guess technically even an ad-serving blog would qualify under. That said, I'm absolutely not intending for small personal hobby blogs, projects, etc. to require a commercial license. It's difficult to quantify this - I.e. is $100 in yearly ad revenue "small" (I'd say yes), what about $1,000 or $10,000? So there's an element of "use your judgement here".

At the end of the day I have no intention of going after, suing, or otherwise enforcing the license. It's there mainly to clarify the expected behavior of users and so that you ask questions exactly like this one so we're all on the same page :).

TL;DR: feel free to use it for your personal blog, even if you're serving up ads and gaining some moderate income from them.
